<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastro</class>
 <widget class="QDialog" name="cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="register_title_screen">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cadastro de produtos Agropecuários</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_product">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string extracomment="Nome">Produto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_description">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Descrição</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="input_sell_price">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="input_buy_price">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="register_title_screen_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R$ Preço compra</string>
   </property>
  </widget>
  <widget class="QLabel" name="register_title_screen_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R$ Preço venda</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Milho, Soja, Algodão...</string>
   </property>
   <property name="text">
    <string>Ra&amp;ção pura</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Formu&amp;lação própria</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Medicamen&amp;tos</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>95</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aves</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>95</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bovinos</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>95</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ovinos</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>95</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Caprinos</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Formulaç&amp;ão industrial</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>O&amp;utros produtos</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>95</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Suínos</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>95</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Uso geral</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Manter dados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="control_screen">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Painel de controle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
